pkg/actors/internal/placement.go

```diff
@@ -306,12 +306,10 @@ func (p *ActorPlacement) closeStream() {
 
 		if p.clientStream != nil {
 			p.clientStream.CloseSend()
-			p.clientStream = nil
 		}
 
 		if p.clientConn != nil {
 			p.clientConn.Close()
-			p.clientConn = nil
```

Comment:
Now that the nil-check is back.... what happens if you call Close twice on the same clientConn?

Reply:
Great question, I've tested it out because I had the same doubt, but there are no panics thanks to the [clientconn check](https://github.com/grpc/grpc-go/blob/master/clientconn.go#L1019-L1024)